specs/client-events/visibility-change.md

```diff
@@ -0,0 +1,19 @@
+# Visibility Change
+
+```js
+
+manager.createEventListener('visibilityChange', async event => {
+    console.info('Is no longer visible!', event.payload)
```

Comment:
this also fires when something is visible again so the example here is potentially misleading
```suggestion
    console.info('Page visibility toggled!', event.payload)
```